assets/js/admin/meta-boxes-product.js

```diff
@@ -451,15 +451,18 @@ jQuery( function( $ ) {
 			security    : woocommerce_admin_meta_boxes.save_attributes_nonce
 		};
 
-		$.post( woocommerce_admin_meta_boxes.ajax_url, data, function() {
-			// Reload variations panel.
-			var this_page = window.location.toString();
-			this_page = this_page.replace( 'post-new.php?', 'post.php?post=' + woocommerce_admin_meta_boxes.post_id + '&action=edit&' );
+		$.post( woocommerce_admin_meta_boxes.ajax_url, data, function( response ) {
+			if ( response.error ) {
+				// Error.
+				window.alert( response.error );
+			} else if ( response.data ) {
+				// Success.
+				$( '.product_attributes' ).html( response.data.html );
+				$( '.product_attributes' ).unblock();
 
-			// Load variations panel.
-			$( '#variable_product_options' ).load( this_page + ' #variable_product_options_inner', function() {
+				// Reload variations panel.
 				$( '#variable_product_options' ).trigger( 'reload' );
-			});
+			}
```

Comment:
I believe a change in this file introduced a bug. I tested in 3.4/master and it doesn't happen there, but I was able to reproduce every time on this branch. Here is how to reproduce:
1. Go to "Add new product" screen.
2. Make the new product a variable product and give it some attributes. Use the attributes for variations. Save the attributes using the Save Attributes button.
3. Observe that the Attributes panel collapses after the save. Previously it would stay open if it was open.
4. Go to the Variations tab.
5. Observe that you get a "Before you can add a variation you need to add some variation attributes on the Attributes tab." message and cannot create variations.

Here is a gif of this: 
![variationsbug](https://user-images.githubusercontent.com/7317227/41738398-cc1bc8e0-7546-11e8-85b7-2bf4bbb0010c.gif)

Reply:
The hiding is ok; but the other is a bug. Patched.